setup.py

```diff
@@ -4,7 +4,7 @@
 if __name__ == '__main__':
     setup(
         name='mirdata',
-        version='0.0.20',
+        version='0.0.21',
```

Comment:
change this right before merging, just in case another PR gets merged before this!